test/dev-app/views/test/elastic.lp

```diff
@@ -0,0 +1,48 @@
+<?lua local form = require "sailor.form" ?>
```

Comment:
I edited it. I convert a nested table into a string. Check https://github.com/sailorproject/sailor/pull/125/commits/2c8be478eca315fe7dc9c97898ef8d5df202ad11